src/core/electrostatics_magnetostatics/mmm-common.hpp

```diff
@@ -19,8 +19,14 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 /** \file
-    modified polygamma functions. See Arnold,Holm 2002
-*/
+ *  Common parts of the MMM family of methods for the electrostatic
+ *  interaction, MMM1D, MMM2D and ELC.  This file contains the code for the
```

Comment:
Double space after "ELC."

Reply:
I think the double whitespace after a dot was a recommendation in the early days of UNIX. But in ESPResSo it is rarely used, except in license headers. I've removed them outside licenses.